engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41500to41510.java

```diff
@@ -235,6 +242,138 @@ public void updateSystemVmTemplates(final Connection conn) {
         LOG.debug("Updating System Vm Template IDs Complete");
     }
 
+    private void correctGuestOsNames(final Connection conn) {
```

Comment:
This corrects the guest os names added in 4.14.0 to 4.15.0 schema.

https://github.com/apache/cloudstack/blob/571339f8724fc70c472f55f254125f0c6070d5de/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L729

https://github.com/apache/cloudstack/blob/571339f8724fc70c472f55f254125f0c6070d5de/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L739

https://github.com/apache/cloudstack/blob/571339f8724fc70c472f55f254125f0c6070d5de/engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql#L805-L807

https://github.com/apache/cloudstack/blob/571339f8724fc70c472f55f254125f0c6070d5de/engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql#L21-L24